From 69deebc1cd8b4ad099bf6550ea2a7ad598f0b7fb Mon Sep 17 00:00:00 2001
From: Sebastian Walderich <sebastian@walderich.de>
Date: Mon, 23 Jul 2012 11:49:30 +0200
Subject: [PATCH] wxTreeCtrl::MSWOnNotify supress reload of comctl32

The loading of the "comctl32.dll" is guarded by the static variable
"loaded". This had to be set to "true" after the loading was successful.
---
 src/msw/treectrl.cpp |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/msw/treectrl.cpp b/src/msw/treectrl.cpp
index e84bd1b..9e99796 100644
--- a/src/msw/treectrl.cpp
+++ b/src/msw/treectrl.cpp
@@ -3513,7 +3513,10 @@ bool wxTreeCtrl::MSWOnNotify(int idCtrl, WXLPARAM lParam, WXLPARAM *result)
                             {
                                 wxLoadedDLL dllComCtl32(wxT("comctl32.dll"));
                                 if ( dllComCtl32.IsLoaded() )
+                                {
                                     wxDL_INIT_FUNC(s_pfn, ImageList_Copy, dllComCtl32);
+                                    loaded = true;
+                                }
                             }
 
                             if ( !s_pfnImageList_Copy )
-- 
1.7.10.msysgit.1

